libs/island-ui/core/src/lib/DialogPrompt/DialogPrompt.treat.ts

```diff
@@ -0,0 +1,69 @@
+import { style } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+
+const hexToRgb = (hex: string) => {
```

Comment:
I've seen this in multiples places, I would move it to utils and export it from islandui

Reply:
You're right. A util like
```
const hexToRgba = (hex: string, alpha: number) => {
  // Requires 6 number hexadecimal
  const rgb = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)

  if (!rgb) {
    return null
  }

  const r = parseInt(rgb[1], 16)
  const g = parseInt(rgb[2], 16)
  const b = parseInt(rgb[3], 16)

  return `rgba(${r}, ${g}, ${b}, ${alpha})`
}
```

Reply:
I would probably re-use where it was defined locally in each projects at the same time

Reply:
I don't want to touch other projects in this particular PR. Will make another for that.